vagrant-pxe-harvester/README.md

```diff
@@ -17,25 +17,22 @@ Prerequisites
 -   vagrant-libvirt plugin \>= 0.0.43.
 -   KVM (i.e. qemu-kvm), preferably the latest and greatest. This
     environment was tested with qemu-kvm 2.11.
--   Host with at least 16GB RAM & 500GB free disk space.
+-   Host with at least 16 CPU, 64GB RAM, and 500GB free disk space.
 
 Quick Start
 -----------
 
 1.  Edit `settings.yml` to make sure the configuration satisfies your
     needs. The options are self-documented.
-2.  Run `setup_harvester.sh`. This may take awhile (i.e. 30 minutes
```

Comment:
I think we should use this one as #15 did.

Reply:
Good catch! Looks rebase didn't pull in that change. Let me retry.

Reply:
It may suffer a bit when using fork on GitHub, maybe u can try like [this example](https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a).

As I know, GitHub's UI only support **fetch-merge** strategy for forked repo.